codec/src/main/java/io/netty/handler/codec/compression/ZlibDecoder.java

```diff
@@ -16,16 +16,72 @@
 package io.netty.handler.codec.compression;
 
 import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.channel.ChannelHandlerContext;
 import io.netty.handler.codec.ByteToMessageDecoder;
 
 /**
  * Decompresses a {@link ByteBuf} using the deflate algorithm.
  */
 public abstract class ZlibDecoder extends ByteToMessageDecoder {
 
+    /**
+     * Maximum allowed size of the decompression buffer.
+     */
+    protected int maxAllocation;
```

Comment:
can this be private/final?

Reply:
I'll make it final but leave it as protected in case a subclass wants to read it for any reason.